packages/babel-core/src/parser/util/missing-plugin-helper.ts

```diff
@@ -1,4 +1,7 @@
-const pluginNameMap = {
+const pluginNameMap: Record<
```

Comment:
Can we mark this object `as const`? So we don't have to add typings at all and TS will infer the typings accordingly.

Reply:
It doesn't seem to work, the object will be written below.